checker/src/main/java/org/checkerframework/checker/signedness/jdk.astub

```diff
@@ -193,6 +202,18 @@ class Arrays {
     public static @PolySigned float[] copyOfRange(@PolySigned float[] original, int from, int to);
     public static @PolySigned double[] copyOfRange(@PolySigned double[] original, int from, int to);
     public static @PolySigned boolean[] copyOfRange(@PolySigned boolean[] original, int from, int to);
+    public static <T> @PolySigned T[] copyOfRange(@PolySigned T[] original, int from, int to);
```

Comment:
Why is this necessary?

Reply:
Yes, I think line 205 is not necessary. Sorry for the mistake.